### ISSUE TITLE
Keep language statistics shown when viewport width is small

### ISSUE THREAD

**Issue by ell1e** [go-gitea/gitea#21389]
### Description

With a too small page width, the language statistics appear to be unreachable: the bar with the colorful language break-up below the "Commits"/"Branches"/"Tags"/"Storage" bar just disappears. With that, both the coarse statistics represented by it, as well as the click area to reach the detailed breakdown, seem to be just gone. I haven't found any other way to reach them, so that seems bad.

Maybe a fix would be to just not have it disappear?

### Gitea Version

1.18.0+dev-421-geaa561145

### Can you reproduce the bug on the Gitea demo site?

Yes

### Screenshots
[Screencast from 10-09-2022 07:40:02 PM.webm](https://user-images.githubusercontent.com/64124388/194771570-87ad4976-c529-4e5f-8428-e5eb663b9c8e.webm)

### How are you running Gitea?

It reproduces on someone else's instance as well as the official demo site, I don't run it myself

**Comment by wxiaoguang** [comment]
It's hidden when width is small. That's done by purpose, unless there is a new UI design.

**Comment by ell1e** [comment]
Hm, but if there is no other way to reach the same content, maybe it just shouldn't be? It doesn't seem like hiding it gains much horizontal screen space anyway (and the site works fine with almost no vertical one since there is obviously a scrollbar), so it seems to me like maybe it's just best left visible.

Or could there be another button or tab added to reach the language statistics on a dedicated page maybe for when this bar isn't shown? Just something to keep the functionality accessible.

**Comment by wxiaoguang** [comment]
I could agree if there would be a better solution which enough people like. Not only the language bar, but also many other elements are hidden when width is small. That's a UI decision in history, and that's what I mean `unless there is a new UI design`.

**Comment by ell1e** [comment]
Surely if an element disappears it should move to some other spot or have some other way of still triggering it? (Like a hamburger menu?) It seems to me like generally just removing functionality *entirely* based on screen size is a bad idea.

**Comment by ell1e** [comment]
Okay, so I think this CSS section should simply be removed:
```
@media (max-width: 767px) {
 .repository .repository-summary .segment.language-stats {
  display:none;
 }
}
```
How it looks like with that section gone:
![Screenshot from 2022-10-16 08-56-39](https://user-images.githubusercontent.com/64124388/196022706-f96cc51f-f78e-40a2-a486-6dbc0444c690.png)

Currently/without that removed:
![Screenshot from 2022-10-16 08-55-55](https://user-images.githubusercontent.com/64124388/196022719-76298dbc-271e-4913-a6e0-b97a56676f6a.png)

**Comment by wxiaoguang** [comment]
It should have been fixed by " Improve "language stats" UI #26968 " (although the UI is not perfect)
